### ISSUE TITLE
Not removing the temporary files when using html option

### ISSUE THREAD

**Issue by JonWatkins** [addyosmani/critical#255]
I am trying to run this on an HTML string, But it does not remove any of the temporary files at the end of the process.

My HTML string contains links to css files that are external resources, and it seems to be dealing with this just fine.

```JavaScript
critical.generate({
    inline: true,
    html: content,
    width: 1300,
    height: 1300,
    extract: false,
    minify: true
  }).then(function (output) {
    // handle output here
  }).catch(function (ex) {
    // handle error here
  })
```

The content variable is a rendered pug file. Im using this as part of a publish/cache process in the server. And it is just filling the server up with the temp files (in the root of the application)

Is there a way i can stop this from happening?

Edit:

On stopping the node server the temp files are all removed, and the process exits with a status of 2 and this error ELIFECYCLE.

**Comment by JonWatkins** [comment]
Took a while to get back here and close this, but I ended up running it in a child process this lets it do exactly what I needed.

**Comment by simon-jouet** [comment]
Hi @JonWatkins,

You closed this issue, but I believe this is still a valid issue and the temporary file should be removed without having to stop node.

**Comment by bezoerb** [comment]
Just released [v1.1.0](https://github.com/addyosmani/critical/releases/tag/v1.1.0) which triggers temp file removal after the critical css is computed

/cc @JonWatkins @simon-jouet
